src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.number = getRandomNumber();
+        ball.color = String.valueOf(new ColorSupplier().getRandomColor());
+        return ball;

```

Comment:
```suggestion
        Ball ball = new Ball();
        ball.number = getRandomNumber();
        ball.color = new ColorSupplier().getRandomColor().name();
        return ball;

```
same lets make  ColorSupplier as class level variable, not to create new instance every time we call getRandomBall() method